### ISSUE TITLE
FTBFS with python-six >= 1.11.0

### ISSUE THREAD

**Issue by frozencemetery** [wichert/pyrad#70]
Hi,

```
rharwood@seton:~/pyrad$ nosetests3
.............................................................................................................................F.....................................................
======================================================================
FAIL: testPktEncodeAttributes (pyrad.tests.testPacket.PacketTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/bos/rharwood/pyrad/pyrad/tests/testPacket.py", line 199, in testPktEncodeAttributes
    six.b('\x1a\x0d\x00\x00\x00\x01\x02\x07value\x01\x07value'))
AssertionError: b'\x01\x07value\x1a\r\x00\x00\x00\x01\x02\x07value' != b'\x1a\r\x00\x00\x00\x01\x02\x07value\x01\x07value'

----------------------------------------------------------------------
Ran 179 tests in 9.085s

FAILED (failures=1)
rharwood@seton:~/pyrad$ 
```

Verified broken with both Debian's python-six-1.11.0-1 and Fedora's python3-six-1.11.0-1.fc28.

**Comment by rbricheno** [comment]
This is a duplicate of https://github.com/wichert/pyrad/issues/68 which is fixed by my PR https://github.com/wichert/pyrad/pull/80

**Comment by GIC-de** [comment]
fixed with PR #80
